src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -1011,6 +1012,16 @@ void Parser::parse_and_visit_export(Parse_Visitor_Base &v,
   switch (this->peek().type) {
     // export default class C {}
   case Token_Type::kw_default:
+    if (this->export_default_statement_span_.has_value()) {
+      this->diag_reporter_->report(
+          Diag_Multiple_Export_Defaults{
+              .second_export_default = this->peek().span(),
+              .first_export_default = *this->export_default_statement_span_,
+          });
+    } else {
+      this->export_default_statement_span_ = this->peek().span();
```

Comment:
Nit: I think the name `export_default_statement_span_` is wrong. The span is not for the whole statement, but for the 'default' keyword. `first_export_default_statement_default_keyword_` is a good (but long) name. (I think length isn't a problem in this case; clarity is more important.)